draft-ietf-quic-http.md

```diff
@@ -384,11 +387,11 @@ client MUST treat this as a connection error of type HTTP_DUPLICATE_PUSH.
 
 ### Reserved Stream Types {#stream-grease}
 
-Stream types of the format `0x1f * N` are reserved to exercise the requirement
-that unknown types be ignored. These streams have no semantic meaning, and can
-be sent when application-layer padding is desired.  They MAY also be sent on
-connections where no request data is currently being transferred. Endpoints MUST
-NOT consider these streams to have any meaning upon receipt.
+Stream types of the format `0x1f * N` for non-zero values of N are reserved to
```

Comment:
Rather than grapple with the 0 value, why not 0x1f*N+0x21 or something like that?

Reply:
Hm, that might be easier.